static/css/marble.css

```diff
@@ -1393,6 +1387,76 @@ h6{
         margin-right: 10px
     }
 
+    /*About*/
+    .margin-committee-title{
+        margin-bottom: 30px;
+    }
+}
+
+@media (width >= 576px) {
```

Comment:
Doesn't this just add back the extra `@media (width >= 576px)` block that you condensed into one in the other PR?

Reply:
Pulled in the >=576px media query changes from fix-node-menu.